cmd/scip/print.go

```diff
@@ -12,31 +13,49 @@ import (
 )
 
 func printCommand() cli.Command {
-	var json bool
+	var json, colorOutput bool
 	snapshot := cli.Command{
 		Name:  "print",
-		Usage: "Print a SCIP index in a human-readable format for debugging",
-		Description: `WARNING: The output may change over time.
-Do not rely on the output of this command in scripts`,
+		Usage: "Print a SCIP index for debugging",
+		Description: `WARNING: The TTY output may change over time.
+Do not rely on non-JSON output in scripts`,
 		Flags: []cli.Flag{
 			&cli.BoolFlag{
 				Name:        "json",
 				Usage:       "Output in JSON format",
 				Destination: &json,
 			},
+			&cli.BoolFlag{
+				Name:        "color",
+				Usage:       "Enable color output for TTY (no effect for JSON)",
+				Destination: &colorOutput,
+				Value:       true,
+				DefaultText: "true",
+			},
 		},
 		Action: func(c *cli.Context) error {
 			indexPath := c.Args().Get(0)
 			if indexPath == "" {
 				return errors.New("missing argument for path to SCIP index")
 			}
-			return printMain(indexPath, json, c.App.Writer)
+			// Following https://no-color.org/
+			if val, found := os.LookupEnv("NO_COLOR"); found && val != "" {
+				switch val {
+				case "":
+					break
+				case "0", "false", "FALSE", "off", "OFF":
```

Comment:
Perhaps it's easier to just lowercase the value before matching

Reply:
Ah yes, done